ramls/traits/queriable.raml

```diff
@@ -4,4 +4,4 @@ queryParameters:
     type: string
     description: String used to search to retrieve a collection
     example: ABC-CLIO
-    required: true
+    required: false
```

Comment:
@dmtkachenko : Just curious, why is this made false? Isn't a search param needed to be able to search a collection ?

Reply:
it can be omitted. for instance you can apply filter but do not use "q" param:
![screenshot_1](https://user-images.githubusercontent.com/42244720/48129072-24c7a880-e291-11e8-8b70-75c9f46f4327.png)

Reply:
👍